src/window.cpp

```diff
@@ -474,6 +474,11 @@ void Window::keyPressEvent(QKeyEvent* event)
         load_next();
         return;
     }
+    else if (event->key() == Qt::Key_Escape)
+    {
+        close();
```

Comment:
I think it might be better to use `QCoreApplication::exit(0)` to more explicitly show the intent of the action. See [this](https://doc.qt.io/qt-5/qcoreapplication.html#exit) for more info.